lib/Sema/AssociatedTypeInference.cpp

```diff
@@ -298,6 +298,12 @@ static void recordTypeWitness(NormalProtocolConformance *conformance,
       aliasDecl->getAttrs().add(attr);
     }
 
+    // Construct the availability of the type witnesses based on the
+    // availability of the enclosing type and the associated type.
+    const Decl * availabilitySources[2] = {dc->getAsDecl(), assocType };
```

Comment:
Whitespace is a bit wonky here

Reply:
Indeed. Fixed up in https://github.com/apple/swift/pull/71512